math/src/field/fields/montgomery_backed_prime_fields.rs

```diff
@@ -470,7 +470,7 @@ mod tests_u384_prime_fields {
         let x = U384F23Element::from(11_u64);
         let x_serialized = serde_json::to_string(&x).unwrap();
         let x_deserialized: U384F23Element = serde_json::from_str(&x_serialized).unwrap();
-        assert_eq!(x_serialized, "{\"value\":\"0xb\"}");
+        // assert_eq!(x_serialized, "{\"value\":\"0xb\"}"); // serialization is no longer as hex string
```

Comment:
should this be removed?

Reply:
[It was previously serialized as a hex string](https://github.com/lambdaclass/lambdaworks/blob/cffa505af7c0c9f73d938577958be20e6b92676a/math/src/field/element.rs#L445), the way it's done now makes the serialized bytes not human readable.

<img width="770" alt="image" src="https://github.com/lambdaclass/lambdaworks/assets/62400508/08f9b520-af96-4102-b043-cf02716e8854">

I assume that the assert was only done to check that the readability was ok, as the next assertion done on the test already verifies that the (de)serealization is done correctly.